### PR TITLE
Add dinov2 model

### PR DESCRIPTION
**Pull request by rajveer43**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Documentation: @stevhliu and @MKhalusova

### CODE REVIEW

File: docs/source/ja/model_doc/dinov2.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+print((last_hidden_states - traced_outputs[0]).abs().max())
+```
+
+## Resources
+
+DPT を始めるのに役立つ公式 Hugging Face およびコミュニティ (🌎 で示されている) リソースのリスト。
+
+- DINOv2 のデモ ノートブックは [こちら](https://github.com/NielsRogge/Transformers-Tutorials/tree/master/DINOv2) にあります。 🌎
+
+<PipelineTag パイプライン="image-classification"/>
```
Comment (stevhliu) [inline docs/source/ja/model_doc/dinov2.md:64]: This should remain untranslated

**Review by stevhliu — APPROVED** [review]
LGTM! Remember not to translate any of the special syntax used by the hf-doc-builder

File: docs/source/ja/model_doc/dinov2.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+# DINOv2
+
+## Overview
+
+DINOv2 モデルは、[DINOv2: Learning Robust Visual features without Supervision](https://arxiv.org/abs/2304.07193) で提案されました。
+Maxime Oquab, Timothée Darcet, Théo Moutakanni, Huy Vo, Marc Szafraniec, Vasil Khalidov, Pierre Fernandez, Daniel Haziza, Francisco Massa, Alaaeldin El-Nouby, Mahmoud Assran, Nicolas Ballas, Wojciech Galuba, Russell Howes, Po-Yao Huang, Shang-Wen Li, Ishan Misra, Michael Rabbat, Vasu Sharma, Gabriel Synnaeve, Hu Xu, Hervé Jegou, Julien Mairal, Patrick Labatut, Armand Joulin, Piotr Bojanowski.
+DINOv2 は、[Vision Transformers](vit) に適用される自己教師ありメソッドである [DINO](https://arxiv.org/abs/2104.14294) のアップグレードです。この方法により、汎用の視覚機能、つまり、微調整することなく画像の配布とタスク全体で機能する機能が可能になります。
+
+論文の要約は次のとおりです。
```
Comment (stevhliu) [inline docs/source/ja/model_doc/dinov2.md:21]: Would you also happen to know of a native Japanese-speaker who can review the content and translation? The [forums](https://discuss.huggingface.co/) or [Discord](https://huggingface.co/join/discord) may be good starting places to find another contributor!

Comment (rajveer43) [reply docs/source/ja/model_doc/dinov2.md:21]: Yeah, I would find one, I was thinking that too

**Pull request closed without merging**